package.json

```diff
@@ -217,7 +217,7 @@
     "eslint": "3.11.1",
     "eslint-plugin-babel": "4.0.0",
     "eslint-plugin-jest": "19.0.1",
-    "eslint-plugin-kibana-custom": "file:packages/custom-eslint-rules",
+    "@elastic/eslint-plugin-kibana-custom": "file:packages/@elastic/eslint-plugin-kibana-custom",
```

Comment:
Yes, in fact, I should do that as part of this PR, which means I'll need to publish this package to npm first. Thanks!